itur/validation/validation_scripts/A_gas_validation.py

```diff
@@ -0,0 +1,335 @@
+# -*- coding: utf-8 -*-
```

Comment:
The latest test scripts produce a simple table with the validation error using the excel spreadsheet you provided. What is the value of these new validation scripts? Also, it seems that you are only validating a subset of all the sheets in the excel spreadsheet.